.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Indeed, we don't want these. Not sure of the best way to get rid of them from git history, so easiest way might be to make a new branch off our `develop`, put the files you changed here that want to include directly in there, and then make a new PR against that branch. We can still discuss things here for now though until then.